openapi.yaml

```diff
@@ -28,35 +28,60 @@ paths:
                   messages:
                     type: "array"
                     items:
-                      type: "object"
-                      properties:
-                        uuid: 
-                          type: "string"
```

Comment:
it's nice to add example data/description if the reader might be unsure what the data looks like